helm-charts/platform-api/templates/platform-api-rbac.yaml

```diff
@@ -3,11 +3,23 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: platform-api-sa
+  labels:
+    name: platform-api-sa
+    app.kubernetes.io/instance: platformapis.operator.ibm.com
```

Comment:
these should be
    app.kubernetes.io/instance: {{ .Release.Name }}
    app.kubernetes.io/managed-by: {{ .Release.Service }}
    app.kubernetes.io/name:  {{ .Values.platformApi.name }}
    helm.sh/chart: {{ .Release.Name }}